woocommerce/google-listings-and-ads
# Streamline campaign setup: Remove ads audience field from paid ads setup during onboarding

### Changes proposed in this Pull Request:



Closes #2501 

* Removed the `AudienceSection` from the `PaidAdsSetupSections` component.
* The `countryCodes` prop in the `paidAds` state variable is now initialized with the values of `targetAudience` which itself contains the values from the second step of the onboarding process.

### Detailed test instructions:


1. Go through the onboarding flow.
2. Note the countries selected in Step 2 in the **Audience** section (**Location**). You can choose specific countries or all countries.
3. On the fourth step of the flow, there should not longer be the "Ads Audience" section. (Refer to the [screenshot in the ticket description](https://github.com/woocommerce/google-listings-and-ads/issues/2501))
4. Create a new paid ad.
5. Go to the dashboard and edit the ad which was just created.
6. Check if the countries displayed match those which were selected in Step 2 of the onboarding process.


### Additional details:


### Changelog entry

> Update - Remove ads audience field from paid ads setup during onboarding.
